docs/content/configuration/storage.md

```diff
@@ -101,7 +101,7 @@ how to create `bucket` inside it.
 
 This driver is using the [AWS S3](https://aws.amazon.com/s3/) and assumes that you already have `account` and `bucket` created in it.
 If you never used Amazon Web Services there is [quick guide](https://docs.aws.amazon.com/AmazonS3/latest/gsg/GetStartedWithS3.html) how to create `bucket` inside it.
-After this you can pass you credentials inside `config.toml` file
+After this you can pass you credentials inside `config.toml` file.  If the access key ID and secret access key are not specified in `config.toml`, the driver will attempt to load credentials for the default profile from the [AWS CLI configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) created during installation.
```

Comment:
could you please fix the typo laying there: your credentials

Reply:
Will do.  Good catch :)